streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamTaskTest.java

```diff
@@ -202,6 +206,8 @@ public void testProcessOrder() throws Exception {
         assertEquals(0, task.numBuffered());
         assertEquals(3, source1.numReceived);
         assertEquals(3, source2.numReceived);
+
+        cleanRun = true;
```

Comment:
I guess my question is: does it matter how we close it in `@After`? If it doesn't change the test result and cleanly shuts down then i don't see why we can't just always pass `true`. If it does impact it then keep it.